.infrastructure/todoapp-pod.yml

```
@@ -1,23 +1,55 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: todoapp
+  name: todoapp-1
   namespace: todoapp
+  labels:
+    app: todolist
 spec:
   containers:
-  - name: todoapp
-    image: ikulyk404/todoapp:3.0.0
-    ports:
-    - containerPort: 8080
-    livenessProbe:
-      httpGet:
-        path: api/health
-        port: 8080
-      initialDelaySeconds: 60
-      periodSeconds: 5
-    readinessProbe:
-      httpGet:
-        path: api/ready
-        port: 8080
-      initialDelaySeconds: 5
-      periodSeconds: 5
+    - name: todoapp-container-1
+      image: ikulyk404/todoapp:3.0.0
+      ports:
+        - containerPort: 8080
+      livenessProbe:
+        httpGet:
+          path: api/health
```

please use path like this 
```suggestion
          path: /api/health
```